docs/reference/workflows/configuring_jobs.rst

```diff
@@ -64,7 +64,19 @@ For example, if a job is defined as follows:
     EXECUTABLE script.sh
     STOP_ON_FAIL TRUE                   -- Tell the job to stop ert on failure
 
-STOP_ON_FAIL can also be specified within the internal (python) or external (executable) job script.
+In case of an external sh job script, you have to manually set the stop on fail option (-e)
+
+::
+    set -e
+
+For example, this bash external job script will stop on failure
+
+::
+    #!/bin/bash
+    set -e
+    ekho 'Hello World' # misspelled command fails
```

Comment:
Explain that we still need `STOP_ON_FAIL TRUE` in ert config.

```suggestion
Shell scripts (Bash) must, in addition to having `STOP_ON_FAIL TRUE` in ert config, set the "exit immediately" flag (`set -e`) for the error to propagate. Otherwise, even if errors occur within the script, it will still be "successful".

::
    #!/usr/bin/env bash
    set -e
    ekho 'Hello World' # misspelled command fails
```